Resources/ServerInfo/Guidebook/Science/Robotics.xml

```diff
@@ -0,0 +1,51 @@
+<Document>
+  # Robotics
+  The science team has the ability to create several unique types of robots to suit the station's needs. In order to begin, you'll want to research [color=#a4885c]Robotics[/color] technology.
+
+  ## Exosuit Fabricator
+  <Box>
+<GuideEntityEmbed Entity="ExosuitFabricator" Caption="Exosuit Fabricator"/>
+</Box>
+  The [color=#a4885c]Exosuit Fabricator[/color] is where you'll be printing out most of the parts for your creations, such as:
+  - Bot limbs
+  - Mech parts and equipment
+  - Proximity sensors
+
+  Operating the exosuit fabricator is the same as every other lathe on the station; simply click the fabricator to open the UI, then click the icon for whatever part you want to print.
+  - Are there no recipes? click the [color=#a4885c]sync[/color] button on the UI to check for any new blueprints. If, even then, nothing shows up, you've likely not researched the relevant technology yet!
+  - If a recipe appears grayed-out, it's likely due to a lack of materials. To insert new materials (either [color=#99A3A4]steel[/color], [color=#ECF0F1]plastic[/color], or [color=#AED6F1]glass[/color]), simply click the fabricator while holding said material.
+
+  Every station starts with a full suite of robotics equipment by default, but you can make another fabricator if [color=#a4885c]Robotics[/color] technology is available, which will allow you to make the respective circuit board.
+
+  ## Bots
```

Comment:
i think the four bots medibot, honkbot and mimebot can fit in the same line/two, add them and if it visually goes for more than that then just add two or three

Reply:
fixed